machine_management/cluster_api_machine_management/cluster-api-getting-started.adoc

```diff
@@ -24,7 +24,7 @@ When you install a cluster that supports managing infrastructure resources with
 * One provider-specific infrastructure cluster resource.
 
 On clusters that support migrating Machine API resources to Cluster API resources, a two-way synchronization controller creates these primary resources automatically.
-For more information, see xref:../../machine_management/cluster_api_machine_management/cluster-api-getting-started.adoc#capi-mapi-migration-overview_cluster-api-getting-started[Migrating Machine API resources to Cluster API resources].
+For more information, see xref:../../machine_management/cluster_api_machine_management/cluster-api-getting-started.adoc#mapi-to-capi-migration-overview_cluster-api-getting-started[Migrating Machine API resources to Cluster API resources].
```

Comment:
Well, you certainly know about links and link text.......

Reply:
yeah, this is allowed - it's not under an additional resources section